### PR TITLE
Use GitHub as the source of documentation

### LINKED ISSUE #7: add readme with plugin wiki URL
$subject

### PR DESCRIPTION
**Pull request by zbynek**
Fixes #7 , see also https://issues.jenkins-ci.org/browse/WEBSITE-637

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+## Release 0.28
+
+* Update to xtrigger-lib 0.8 (fix
+[JENKINS-12888](https://issues.jenkins-ci.org/browse/JENKINS-12888))
+
+## Release 0.27
+
+* Fix
+[JENKINS-12865](https://issues.jenkins-ci.org/browse/JENKINS-12865) -
+[ERROR](https://wiki.jenkins.io/display/JENKINS/FSTrigger+Plugin#) -
```
Comment (nlk35) [inline CHANGELOG.md:78]: You can replace https://wiki.jenkins.io/display/JENKINS/FSTrigger+Plugin# with https://plugins.jenkins.io/fstrigger/
as it is already redirecting to later url

File: README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
+# FSTrigger Plugin for Jenkins
```
Comment (oleg-nenashev) [inline README.md:1]:
```suggestion
# Filesystem Trigger Plugin for Jenkins
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/fstrigger.svg)](https://plugins.jenkins.io/fstrigger)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/fstrigger.svg?color=blue)](https://plugins.jenkins.io/fstrigger)
+
+FSTrigger provides polling mechanisms to monitor a file system and
```
Comment (oleg-nenashev) [inline README.md:6]:
```suggestion
Filesystem Trigger Plugin (also known as FSTrigger) provides polling mechanisms to monitor a file system and
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+The plug-in makes it possible to monitor changes of a file or a set of
+files in a folder.
+
+![](docs/images/fstrigger.png)
+
+Note: The plug-in uses only persistence in memory. There is no impact on
+the Jenkins infrastructure (no new files created).  
+This plugin provides a polling typology among the [XTrigger
+Plugin](https://wiki.jenkins.io/display/JENKINS/XTrigger+Plugin).
```
Comment (oleg-nenashev) [inline README.md:19]:
```suggestion
Plugin](https://github.com/jenkinsci/xtrigger-plugin).
```

**Comment by zbynek** [comment]
@nlk35 @oleg-nenashev thanks for the suggestions, I committed all of them.

**Comment by zbynek** [comment]
@TonyNoble could you please check this?

**Pull request merged**